packages/oui-navbar/_mixins.less

```diff
@@ -369,6 +375,47 @@
         outline-color: initial;
         outline-offset: -2px;
       }
+
+      .notification__icon {
+        margin: rem-calc(3) rem-calc(10) 0 0;
+      }
+
+      .notification__icon > i {
```

Comment:
Careful on using `i` tag, since it could be replaced by a `span`
Prefer using a classname like `.oui-icon` instead.

And it could be exported too the selector below :)